src/fsa_anno.cc

```diff
@@ -448,6 +448,50 @@ FsaAnno FsaAnno::literal(LiteralExpr& expr) {
   return r;
 }
 
+FsaAnno FsaAnno::case_insensitive_literal(CaseInsensitiveLiteralExpr& expr) {
+  FsaAnno r;
```

Comment:
在 `fsa_anno.cc` 的 `case_insensitive_literal` 方法中，对于非字母字符，直接复制了原始字符到转换表中。考虑到Unicode字符的情况，是否应该对所有非ASCII字符都进行相同处理，或者需要特殊处理？

```suggestion
// 可以考虑添加注释说明非ASCII字符的处理逻辑
// 或者为非ASCII字符添加特殊处理（如果需要的话）